src/doc/doc-builders.js

```diff
@@ -33,6 +53,10 @@ function indent(contents) {
   return { type: "indent", contents };
 }
 
+/**
+ * @param {number} n
```

Comment:
This change would resolve the type error ignored in `markAsRoot` below:

```suggestion
 * @param {number | Doc} n
```

but I am not sure if this is something we really want to do here.

I wonder if we should consider adding a special align function to get what we need for `markAsRoot` instead?